Makefile

```diff
@@ -13,23 +15,33 @@ CST_CERTS_DIR ?= .certs
 
 .PHONY: build get-dev-deps lint test test-with-coverage
 		generate-self-signed-certificate
+		security-check
+		validate-swagger-spec
 
 build:
 	$(GO) build -o "$(CSTBIN)"
 
-test: lint test-with-coverage
+test: lint security-check test-with-coverage validate-swagger-spec
 
 lint:
 	$(GOLINT) $(shell $(GO) list ./...)
 
+security-check:
+	$(GOSEC) -severity medium ./...
+
 test-with-coverage:
 	$(GO) test -v -cover -coverprofile=$(COVERAGE_FILE) ./...
 	grep -v "mock.go" $(COVERAGE_FILE) > coverage.txt
 	$(GO) tool cover -func=coverage.txt
 
+validate-swagger-spec:
+	$(SWAGGER) validate swagger.yml
+
 get-dev-deps:
-	$(GO) get -u golang.org/x/lint/golint
 	$(GO) get -u github.com/golang/dep/cmd/dep
+	$(GO) get -u golang.org/x/lint/golint
```

Comment:
I suggest that you use https://github.com/golangci/golangci-lint

Reply:
:+1:

Reply:
It's a really good suggestion. I'll replace golint in another PR.